### PR TITLE
DOP-2824: Search results include visual indicator of source

### PR DESCRIPTION
**Pull request by gjchong25**
### Stories/Links:

[DOP-2824](https://jira.mongodb.org/browse/DOP-2824)

### Current Behavior:

[Landing Staging Current](https://docs-mongodbcom-integration.corp.mongodb.com/master/landing/raymundrodriguez/DOP-1998/search/?q=mongo)

### Staging Links:

[Landing Staging](https://docs-mongodbcom-integration.corp.mongodb.com/master/landing/grace.chong/DOP-2824/search/?q=mongo)

### CODE REVIEW

**Review by rayangler — COMMENTED** [review]
Looks good so far! Just some questions and nits for now

File: src/components/Searchbar/SearchResult.js [hunk 1]
```diff
@@ -179,6 +205,11 @@ const SearchResult = React.memo(
               __html: sanitizePreviewHtml(highlightedPreviewText),
             }}
           />
+          <StylingTagContainer>
+            {!!category && <StyledTag variant="green">{category}</StyledTag>}
+            {!!version && <StyledTag variant="blue">{version}</StyledTag>}
+            {(url.includes('/api/') || url.includes('/graphql/')) && <StyledTag variant="purple">{'API'}</StyledTag>}
```
Comment (gjchong25) [inline src/components/Searchbar/SearchResult.js:211]: As per Jon's comment [here](https://jira.mongodb.org/browse/DOP-2824), GraphQL API content does not have 'api' as part of the URL, so it was added as an additional check.

File: src/components/Tag.js [hunk 2]
```diff
@@ -23,6 +30,11 @@ const VARIANTS = {
     border: 1px solid ${uiColors.green.light2};
     color: ${uiColors.green.dark2};
   `,
+  purple: css`
+    background-color: ${palette.purple.light3};
```
[the remark this answers is not in the document]
Comment (rayangler) [reply src/components/Tag.js:34]: Ah, sorry, the other variants still use `uiColors`, which should be the old colors. If we use `palette` like what's done with purple, the other variants should have their new colors!

File: src/components/Searchbar/SearchResult.js [hunk 3]
```diff
@@ -110,12 +123,28 @@ const sanitizePreviewHtml = (text) =>
...
+    ...props
+  }) => {
     const { searchContainerRef, searchTerm } = useContext(SearchContext);
     const highlightedTitle = highlightSearchTerm(title, searchTerm);
     const highlightedPreviewText = highlightSearchTerm(preview, searchTerm);
     const resultLinkRef = useRef(null);
 
+    const { filters } = useMarianManifests(true);
+    const category = !!categoryTag ? categoryTag : filters?.[searchProperty]?.['category'];
+    const version = !!versionTag ? versionTag : filters?.[searchProperty]?.['version'];
```
[the remark this answers is not in the document]
Comment (rayangler) [reply src/components/Searchbar/SearchResult.js]: Cool, thanks for the explanation! Whether we choose to keep it or not, I guess we'll just have to keep an eye on things while we continue improving our manifests

**Review by rayangler — COMMENTED** [review]
Mostly LGTM! Just one more follow-up comment about the Tag colors, if we want to get the new colors instead of the old

File: src/components/Tag.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (gjchong25) [reply src/components/Tag.js:34]: Oh sorry, misunderstood your comment! Fixing now

File: src/components/Searchbar/SearchResult.js (hunk 1 above)
Comment (gjchong25) [reply src/components/Searchbar/SearchResult.js]: After clarification, it was decided that GraphQL API content should not be tagged as API.

**Review by rayangler — APPROVED** [review]
LGTM!

**Review by allisonmui9 — APPROVED** [review]
very nice lgtm!

**Pull request merged**